### PR TITLE
[DEV-4751] goal templates read

### PR DESCRIPTION
**Pull request by gaetano-guerriero**

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -10509,3 +10570,45 @@ components:
...
+      type: object
+      additionalProperties: false
+      properties:
+        id:
+          description: Identifier of the template.
+          type: integer
+        name:
+          $ref: '#/components/schemas/GoalTemplateName'
+        metric:
+          # {% if False %}
```
Comment (vmarkovtsev) [inline openapi.yaml:10596]:
```suggestion
# {% if False %}
```

File: openapi.yaml [hunk 2]
```diff
@@ -10509,3 +10570,45 @@ components:
...
+          allOf:
+# {% else %}{{ "\n" }}          oneOf:{% endif %}
+          - $ref: '#/components/schemas/PullRequestMetricID'
+          - $ref: '#/components/schemas/JIRAMetricID'
+          - $ref: '#/components/schemas/ReleaseMetricID'
+      required:
+      - id
+      - name
+      - metric
```
Comment (vmarkovtsev) [inline openapi.yaml:10605]: Same here, alphabetical sort. I wonder if the check can be automated...

**Review by vmarkovtsev — CHANGES_REQUESTED** [review]

File: openapi.yaml (hunk 2 above)
Comment (gaetano-guerriero) [reply openapi.yaml:10610]: Interesting, yaml I think does not require order while reading keys so it is not immediate.

Comment (vmarkovtsev) [reply openapi.yaml:10610]: This is more like Black for Yaml

**Review by vmarkovtsev — APPROVED** [review]

**Pull request merged**